src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/CalculableSentence.kt

```diff
@@ -188,6 +188,9 @@ interface CalculableSentence {
 
   fun calculateErsedFromHalfway(): Boolean
   fun calculateErsedFromTwoThirds(): Boolean
+  fun calculateMixed(): Boolean {
+    return false
+  }
```

Comment:
minor but could replace with just one line
`fun calculateMixed(): Boolean = false`

Reply:
Also should probably be named related to ERSED.

Reply:
Nice comments - doing now.